### PR TITLE
fix: isDeleted added to MetaDataType

### LINKED ISSUE #72: Add isDeleted status to MetaDataType
The original `MetaDataType` specification was based on that developed by [ICAR](https://github.com/adewg/ICAR), which has since been extended by the addition of an `isDeleted` boolean field. Adding `isDeleted` (not required, default False) will allow systems to synchronise data more effectively, including marking records as deleted.

### PR DESCRIPTION
**Pull request by donwilson-rezare**
isDeleted boolean has been added to MetaDataType.json

### CODE REVIEW

**Review by daniel-buchanan — COMMENTED** [review]
@cookeac we realised that this was not present (and there was no way to tell if anything using this schema had been deleted), so I suggested to @donwilson-rezare that this would be helpful.

File: types/MetaDataType.json [hunk 1]
```diff
@@ -36,6 +36,10 @@
         "$ref": "../types/DateTimeType.json",
         "description": "The latest date and time to which the object is valid.",
         "nullable": true
+      },
+      "isDeleted": {
+        "type": "boolean",
```
Comment (cookeac) [inline types/MetaDataType.json:37]:
```suggestion
        "type": "boolean",
        "default": false,
```

**Review by cookeac — CHANGES_REQUESTED** [review]
1. I've added an issue #72 because changes should be accompanied by a discussion thread. Please link the PR to the issue.
2. I've suggested adding "default": false purely as documentation. Default does not populate values, it just tells validators (and readers) that missing values have the same meaning (in this case, if not specified, `isDeleted` is false. See https://json-schema.org/understanding-json-schema/reference/generic.html

**Comment by donwilson-rezare** [comment]
@cookeac  I realised after I saw your suggested change that I had not merged my previous PR to change all the "nullable" references. 

I have addressed this now and resolved the conflict.  Please approve again.

**Comment by cookeac** [comment]
@donwilson-rezare  and now for some reason it shows "default" on two lines - so maybe need to reverse out my suggestion (the other line didn't show up before when I reviewed, but it must have been there).

**Comment by donwilson-rezare** [comment]
That was my fault, Andrew.  Very sorry.

**Review by cookeac — APPROVED** [review]

**Pull request merged**